File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1115.rb
while(true) do
    ns = gets.chomp().split(' ')
    x = ns[0].to_i
    y = ns[1].to_i
    if(x==0 or y==0) then
        break
    end
    if(x>0) then
        if(y>0) then
    		puts("primeiro");
        else
    		puts("quarto");
        end
    else
        if(y>0) then
    		puts("segundo");
        else
            puts("terceiro");
        end
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1115.cs
using System;

class URI
{
    static void Main() {
        
        int x, y;
        while(true){
            string n = System.Console.ReadLine().Trim();
            string[] ns = n.Split(' ');
            x = int.Parse(ns[0]);
            y = int.Parse(ns[1]);
            if(x==0 || y==0){
                break;
            }
            if(x>0){
                if(y>0){
    				Console.Write("primeiro\n");
                } else{
    				Console.Write("quarto\n");
                }
            } else{
                if(y>0){
    				Console.Write("segundo\n");
                } else{
                    Console.Write("terceiro\n");
                }
            }
        }
    
    }
}